.dtproject

```diff
@@ -1,3 +1,3 @@
 VERSION=4.2.3
-TYPE=dt-ros-commons
+TYPE=template-ros
```

Comment:
This needs to be reverted to "dt-ros-commons", the issues is in the shell not being updated